subsys/net/lib/nrf_cloud/coap/cddl/nrf_cloud_coap_ground_fix.cddl

```diff
@@ -73,10 +73,9 @@ wifi = 20
 ; Ground Fix Response
 ; *******************
 
-methods = MCELL/SCELL/WIFI
 
 ground_fix_resp = {
-	fulfilledWith => methods,
+	fulfilledWith => tstr,
```

Comment:
I thought about this one, and because this is a response from the cloud side, it is not easy for the cloud to send a numeric response for fullfilledWith instead of a string response.  How would it know to do that?

It is not worth the bother to add a special mechanism for numeric vs. text response to the backend just to save 3 bytes in the response.  So I'll accept this change.